### DISCUSSION TITLE
I have successfully released a new version of the package, but it is not available in the NPM repository

### DISCUSSION THREAD

**Discussion by devcui** [npm/feedback#917] [CLI] [answered]
![image](https://user-images.githubusercontent.com/26159886/234207662-88d87f65-ec63-4c40-97f0-dde471cd82e4.png)

![image](https://user-images.githubusercontent.com/26159886/234207718-123801eb-7c98-4955-b37f-6da560d82c39.png)

But when I click to enter one of the packages to view the version
![image](https://user-images.githubusercontent.com/26159886/234207767-50d13233-36f1-4bb9-a1a0-39d8d2f3a5ea.png)

**Comment by devcui** [comment]
When I try to install directly

![image](https://user-images.githubusercontent.com/26159886/234207988-bcf40c75-dd81-4a10-a7cc-dcc06be08640.png)

**Comment by noahziheng** [comment]
I also found same problem with `@artus/core`. These version has published and received success emails, but cannot find in registry and website:

2.0.0-alpha.1
2.0.0-alpha.2
2.0.0-alpha.3
2.0.0-alpha.4
2.0.0-beta.0

**Reply by devcui** [reply]
https://status.npmjs.org/

I found a link that identifies the status of NPM, and I don't know if it is related to the content provided inside

Apr 24, 2023
[Partial outage of package publish](https://status.npmjs.org/incidents/vfwgk7xl1yhn)
Resolved - This incident has been resolved.
Apr 24, 20:04 UTC
Monitoring - A fix has been implemented and we are monitoring the results.
Apr 24, 18:22 UTC
Identified - The issue is being identified and we are working on the fix
Apr 24, 17:49 UTC
Update - We continue to see extremely delayed package publishes and continue to investigate.
Apr 24, 16:02 UTC
Investigating - We have received multiple reports of package publish not working as expected and are investigating.
Apr 24, 15:11 UTC

**Reply by devcui** [reply]
![437C7570](https://user-images.githubusercontent.com/26159886/234213994-2011d77f-02a4-4f8f-a49f-da0fb40a61e7.png)

**Comment by devcui** [comment]
It looks normal again now, possibly due to a delay issue, but the delay time is also too long, about two hours or so

**Comment by zhiqingchen** [comment]
Facing the same problem

**Comment by MylesBorins** [comment, marked as answer]
This is now fixed
